app/core/ts/models/Wire.ts

```diff
@@ -50,13 +50,25 @@ export abstract class Wire extends CullableObject {
             const pos = this.p1.getWorldTargetPos();
             const dir = this.p1.getWorldDir();
             this.shape.setP1(pos);
-            this.shape.setC1(dir.scale(DEFAULT_SIZE).add(pos));
+            // To create a straight bezier curve, c1 must be at the same position as p1
+            if(this.straight) {
+                this.shape.setC1(pos);
+            }
+            else {
+                this.shape.setC1(dir.scale(DEFAULT_SIZE).add(pos));
+            }
```

Comment:
Fix formatting please:
space between `if` and parenthesis,
no brackets needed since it's a one-liner